### ISSUE TITLE
RescriptReactRouter into a separate package?

### ISSUE THREAD

**Issue by tom-sherman** [rescript-lang/rescript-react#33]
It seems strange that it exists inside of these bindings, maybe they should be split out?

**Comment by rickyvetter** [comment]
Yeah. I feel ya. Probably - but it's lower priority than getting the React part of this repo into better shape (which has obviously been pretty low priority for me - but I'm trying to get back into it). If someone wants to do this work I'm happy to provide guidance and get it done. If it's on me to break out, this will probably not happen for a while.

**Comment by tom-sherman** [comment]
@rickyvetter I'd be happy to make a start. Do you have a rough outline of the work that needs to be done to enable this?
